openedx/core/djangoapps/catalog/management/commands/cache_programs.py

```diff
@@ -24,18 +30,107 @@ class Command(BaseCommand):
     help = "Rebuild the LMS' cache of program data."
 
     def handle(self, *args, **options):
-        catalog_integration = CatalogIntegration.current()
-        username = catalog_integration.service_username
+        if not waffle.switch_is_active("populate-multitenant-programs"):
```

Comment:
Minor: here and everywhere else, please stay consistent with the strings already present and use single quotes.